content/docs/install/completion.md

```diff
@@ -31,8 +31,8 @@ Depending on what you typed on the command line so far, it completes:
 - Options (flags) that are available for a particular command.
 - File names that make sense in a given context, such as using them as a target
   for some commands.
-- Values for certain command arguments. For example, `dvc repro` completes with
-  stage files to reproduce.
+- Values for certain command arguments and subcommands. For example, `dvc plots`
+  completes with `diff`, `modify`, and `show`.
```

Comment:
Hmmm I think this should stay as just "command arguments" and find another example. In fact you can keep `repro` just replace "stage file" for the updated term (see https://dvc.org/doc/command-reference/repro to figure that one out 😉 )

Reply:
It's a good idea to mention "subcommands" and maybe even the `plots` example too though, but in the first bullet "Available DVC commands."

Reply:
This makes sense, I moved the `plots` example to the first bullet as you suggest. I restored the `repro` example with updated terminology.

When started working on this the first thing I tested out was the completion for `repro`. It didn't show me available stages. I assumed this meant it was changed, but it might just be a problem with my example project.

Reply:
OK. I see you mentioned trying the tool here. Yeah it's not that smart probably. You can just say something like "existing files" here.